### PR TITLE
Modify methods used in simpleCorsResourcePolicy

### PR DESCRIPTION
**Pull request by akilegaspi**
I find it that most people build APIs when using wai-cors hence modifying the request to add the other methods essential.

### CODE REVIEW

File: src/Network/Wai/Middleware/Cors.hs [hunk 1]
```diff
@@ -251,14 +251,15 @@ data CorsResourcePolicy = CorsResourcePolicy
...
+    , corsMethods = simpleCorsMethods
     , corsRequestHeaders = []
     , corsExposedHeaders = Nothing
     , corsMaxAge = Nothing
     , corsVaryOrigin = False
     , corsRequireOrigin = False
     , corsIgnoreFailures = False
-    }
+    } where
+  simpleCorsMethods = ["HEAD", "GET", "PUT", "POST", "DELETE", "OPTIONS"]
```
Comment (larskuhtz) [inline src/Network/Wai/Middleware/Cors.hs:262]: The [CORS standard](http://www.w3.org/TR/cors/) allows only `GET`, `HEAD`, and `POST` for *simple cross-origin requests*.

The policy that is provided by this function qualifies the resource for *simple cross origin requests*.

A server is free to define other CORS policies for the resources that it serves, but those don't necessarily qualify for *simple cross origin requests* as defined in the standard.

**Review by larskuhtz — COMMENTED** [review]
The proposed change for `simpleCorsResourcePolicy` would result in a CORS policy that doesn't meet the requirements for *simple cross origin requests* as defined by the CORS standard.

**Comment by larskuhtz** [comment]
You are free to use the proposed CORS policy when serving your APIs, but it would require the client to make preflight requests and would thus not be a policy for *simple cross origin requests*. Since this policy would be specific to your API, I think, it is outside the scope of this package.

**Comment by akilegaspi** [comment]
Thank you for the clarification @larskuhtz

**Comment by larskuhtz** [comment]
Thanks for contributing to the package!

**Pull request closed without merging**